### DISCUSSION TITLE
Buttons STAKE and CLAIM not working

### DISCUSSION THREAD

**Discussion by dbwebapp** [BitCannaGlobal/WeedWallet#79] [General]
the 2 buttons below are not working :
![Screenshot 2023-05-24 at 19 45 44](https://github.com/BitCannaGlobal/WeedWallet/assets/96944913/70345967-5dbf-431f-8e41-03fb2c57f83b)

**Comment by TJ-BitCanna** [comment]
Thanks for noticing!

I will close this issue, since it is already reported by @kjincgroup on https://github.com/BitCannaGlobal/WeedWallet/discussions/58.
